nursery/check-for-vm-using-instruction-vpcext.yml

```diff
@@ -0,0 +1,21 @@
+rule:
+  meta:
+    name: check for VM using instruction VPCEXT
+    namespace: anti-analysis/anti-vm/vm-detection
+    authors:
+      - richard.weiss@mandiant.com
+    description: Detects virtualization using VPCEXT (visual property container extender) instruction. Execution of this instruction will cause an illigal instruction exception outside of a virtual environment otherwise return 0
```

Comment:
```suggestion
    description: Detects virtualization using VPCEXT (visual property container extender) instruction. Execution of this instruction will cause an illegal instruction exception outside of a virtual environment otherwise return 0
```